corehq/apps/export/templates/export/download_data_files.html

```diff
@@ -0,0 +1,130 @@
+{% extends "style/base_section.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+
+{% block title %}{% trans "Download Data Files" %}{% endblock %}
+
+{% block page_content %}
+    {% block main_column_content %}
+
+<h2>{% trans "Download Data Files" %}</h2>
+
+{% if data_files %}
+<div class="row">
+    <div class="col-sm-9 col-md-8 col-lg-8">
+        <table class="table table-hover">
+            {% for data_file in data_files %}
+            <tr id="row_{{ data_file.id }}">
+                <td>
+                    <a href="{% url 'download_data_file' domain data_file.id data_file.filename %}">
+                        <i class="fa fa-file text-primary"></i>
+                        {{ data_file.filename }}
+                    </a>
+                </td>
+                <td>
+                    {{ data_file.description }}
+                    <textarea id="url_{{ data_file.id }}" style="display: none">{% if request.is_secure %}https://{% else %}http://{% endif %}{{ request.get_host }}{% url 'download_data_file' domain data_file.id data_file.filename %}</textarea>
+                </td>
+                <td style="width: 6em; text-align: right;">
+                    <a href="#"
+                       onclick="(function () {
+                                    var showCopyDialog = function () {
+                                        var url = '{% if request.is_secure %}https://{% else %}http://{% endif %}{{ request.get_host }}{% url 'download_data_file' domain data_file.id data_file.filename %}';
+                                        window.prompt('{% trans 'Copy to clipboard: Ctrl-C, Enter (Mac: Cmd-C, Enter)' %}',
+                                                      url);
+                                    };
+                                    try {
+                                        // Most browsers since Sept 2015
+                                        var urlElem = $('#url_{{ data_file.id }}');
+                                        urlElem.show();
+                                        urlElem.select();
+                                        var copied = document.execCommand('copy');
+                                        urlElem.hide();
+                                        if (copied) {
+                                            alert_user('{% trans 'Data file URL copied to clipboard.' %}', 'success', true);
+                                        } else {
+                                            showCopyDialog();
+                                        }
+                                    } catch (err) {
+                                        // Older browsers
+                                        showCopyDialog();
+                                    }
+                                }())">
```

Comment:
would be nice to extract this into a separate JS file if you have a chance. same with css